packages/owl-bot/src/owl-bot.ts

```diff
@@ -141,6 +141,23 @@ export = (privateKey: string | undefined, app: Probot, db?: Db) => {
         },
         context.octokit
       );
+      // If running post-processor has created a noop change, close the
+      // pull request:
+      const files = (
+        await context.octokit.pulls.listFiles({
+          owner,
+          repo,
+          pull_number: context.payload.pull_request.number,
+        })
+      ).data;
```

Comment:
One concern is the metadata synth metadata change that often is the only change in the PR. Will that still trigger a PR?

Reply:
@SurferJeffAtGoogle mind speaking to this? We don't believe we actually maintain a synth metadata any longer. Instead  we track what commits haven't yet been applied from `googleapis-gen`.

Reply:
If it goes away, that's great and resolves the concern :)